protocols/v2/noise-sv2/src/initiator.rs

```diff
@@ -107,7 +107,7 @@ impl Initiator {
             ck: [0; 32],
             h: [0; 32],
             e: Self::generate_key(),
-            pk,
+            expected_responder_pk: pk,
```

Comment:
I would add _authority_ here to distinguish from the static pubkey.

Reply:
`expected_authority_pk`, or `expected_responder_authority_pk`?

Reply:
Could also call it `responder_authority_pk`